src/main/java/net/consensys/wittgenstein/core/geoinfo/CityGeoInfoAllCities.java

```diff
@@ -0,0 +1,83 @@
+package net.consensys.wittgenstein.core.geoinfo;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import static net.consensys.wittgenstein.core.Node.MAX_X;
+import static net.consensys.wittgenstein.core.Node.MAX_Y;
+
+public class CityGeoInfoAllCities implements CityGeoInfo {
+
+  private final Map<String, int[]> citiesPosition;
+  private final double mapWidth;
+  private final double mapHeight;
+  private final static Path CITY_PATH = Paths.get("resources/cities.csv");
+
+
+  public CityGeoInfoAllCities() {
+    this.mapWidth = MAX_X;
+    this.mapHeight = MAX_Y;
+    this.citiesPosition = readCityInfo(CITY_PATH);
+  }
+
+  public Map<String, int[]> citiesPosition() {
+    return new HashMap<>(citiesPosition);
+  }
+
+  private Map<String, int[]> readCityInfo(Path path) {
+    Map<String, int[]> citiesPosition = new HashMap<>();
+
+    try (Reader reader = Files.newBufferedReader(path);
+        CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader())) {
+
+      for (CSVRecord csvRecord : csvParser) {
+        String cityName = csvRecord.get(0).replace(' ', '+').toLowerCase();
+        float latitude = Float.valueOf(csvRecord.get(1));
+        float longitude = Float.valueOf(csvRecord.get(2));
+        int mercX = convertToMercerX(longitude);
+        int mercY = convertToMercerY(latitude);
+
+        citiesPosition.put(cityName, new int[] {mercX, mercY});
+      }
+    } catch (IOException e) {
+      throw new IllegalStateException(e);
+    }
+    return citiesPosition;
+  }
+
+  private int convertToMercerX(double longitude) {
```

Comment:
It's really Mercer? I don't see it in the list here for example: https://en.wikipedia.org/wiki/List_of_map_projections
Could you add a link to the projection in the javadoc?